config/webpack.production.config.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path')
+const VueLoaderPlugin = require('vue-loader/lib/plugin')
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin')
+
+const config = {
+  entry: [
+    path.resolve('./src/main.js')
+  ],
+  optimization: {
```

Comment:
AFAICT the only difference between production and development are the `optimization` option, and the `MiniCSSExtractPlugin`.

Since the difference is so minimal, they'll be better placed in a single config file, using ternary operators, e.g.
```js
{
  optimization: process.env.NODE_ENV === 'production'
    ? {...}
    : undefined
}
```

Reply:
if you wish I would do it, but I like the idea to have separate files for better readability and future changes